Build.cmake

```diff
@@ -93,20 +94,32 @@ ENDIF()
    file(GLOB SRC_FILES ${LOG_SRC}/g3log/*.h ${LOG_SRC}/g3log/*.hpp ${LOG_SRC}/*.cpp ${LOG_SRC}/*.ipp)
    file(GLOB HEADER_FILES ${LOG_SRC}/g3log/*.hpp ${LOG_SRC}/*.hpp)
    
+   list( APPEND HEADER_FILES ${GENERATED_G3_DEFINITIONS} )
```

Comment:
This is only needed so the file shows up in IDEs (SRC_FILES) and to make sure it is installed later where you use HEADER_FILES to install the includes.